pkgdb2/templates/list_actions.html

```diff
@@ -123,10 +123,11 @@
         </a>
       </td>
       <td>
-        {% if action.package %}
-          <a href="{{url_for('.package_info', package=action.package.name)}}">
-            {{ action.package.name }}
-          </a>
+        {% if action.package or (action.package == True and request.action == 'request.package'
```

Comment:
``action.package`` will never be ``==True``, it cannot by definition, which means the entire second part of the condition will always fail.